axolotl-web/src/layouts/Default.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class="base-layout">
+    <header :class="currentRoute + ' header fixed-top'">
+      <div class="container">
+        <div class="header-row row">
+          <div class="col-2 header-left">
+            <button
+              v-if="$route.meta.hasBackButton"
+              class="btn"
+              @click="back()"
+            >
+              <font-awesome-icon icon="arrow-left" />
+            </button>
+          </div>
+          <div class="col-8 header-center">
+            <slot name="header" />
+          </div>
+          <div class="col-2 header-right justify-content-end d-flex">
+            <div class="dropdown">
+              <button
+                id="dropdownMenuButton"
+                class="btn btn-primary"
+                type="button"
+                data-bs-toggle="dropdown"
+                aria-expanded="false"
+              >
+                <font-awesome-icon icon="ellipsis-v" />
+              </button>
+              <ul class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+                <slot name="menu" />
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </header>
+    <main class="container">
+      <slot />
+    </main>
+    <footer>
+      <slot name="footer" />
+    </footer>
+  </div>
+</template>
+<script>
+export default {
+  name: "DefaultLayout",
+  data: () => ({
+    showMenu: false,
+  }),
+  computed: {
+    currentRoute() {
+      return this.$route.name;
+    },
+  },
+  methods: {
+    back() {
+      this.$router.go(-1);
```

Comment:
Looking at the vue-router doc, I understand that another option would be `this.$router.back()` - this would align slightly more with the methods function name, and arguably is simpler. Does using `.go()` provide any advantage over `.back()`?

https://router.vuejs.org/api/#back